### ISSUE TITLE
.download() never return

### ISSUE THREAD

**Issue by justshot** [pytube/pytube#307]
I downloaded an audio stream using .download() method, which never return.
It was a 3 min length piece. I killed the download process after some time. The downloaded piece was more than 10 min. The first 3 min is the correct content. The remaining min were silent content. 

Any idea?

**Comment by 09jlardinois** [comment]
I came across that too with 

`YouTube('https://www.youtube.com/watch?v=XLXSqan3Jcs').streams.get_by_itag(137).download( )`

**Comment by 09jlardinois** [comment]
I fixed it. Actually it was never broken. It just was taking a long time to download and I was stupid and impatient. 

In python (Python 3), I used the code: 

`def getbytes(stream, chunk, file_handle, bytes):
    print (int(bytes / 1000000))
    return 0
yt.register_on_progress_callback(getbytes)`

to show the remaining bytes to get a sense of progress. Or megabytes, rather. 

Sorry about the code formatting. I can never get it to work here.
